browser-tests/utils/login.ts

```diff
@@ -10,13 +11,33 @@ const givePermission = async (t: TestController) => {
 };
 
 export const login = async (t: TestController) => {
-  await t
-    .click(ssoLogin.loginLink)
-    .typeText(ssoLogin.username, testUsername())
-    .typeText(ssoLogin.password, testUserPassword())
-    .click(ssoLogin.loginButton);
+  // @ts-ignore
+  const developmentMode = t.testRun.opts.developmentMode
```

Comment:
`--dev` is not a bad idea but I also wonder if just to use another `.env` flag variable like `GITHUB_LOGIN=True` 
or `BROWSER_TESTS_LOCAL = True`. Then you can even run the browser tests against staging or production by changing env parameters. Now you have to have also remove the `--dev` from the yarn script if you want to run tests against some other environment.

Reply:
I think '--dev' -option do not affect to tests itself or limit used environments. 
This is used to check if tests are run by user (use github login) or CI automation (use helsinki tunnus login). 
  
testcafe documentation:
https://testcafe.io/documentation/402638/reference/configuration-file#developmentmode

_**developmentMode**_
_Enables mechanisms to log and diagnose errors. Enable this option if you plan to contact TestCafe Support to report an issue._

Reply:
Keep as it is now